### DISCUSSION TITLE
How to use Bigquad Deck?

### DISCUSSION THREAD

**Discussion by geun-woong** [bitcraze/discussions#818] [Q&A]
Hello!
I made a quadcopter using the Bigquad deck, and even uploaded the firmware with information obtained through another discussion.
In the console of the client, messages such as [BIGQUAD: Switching to brushless.] and [SUP: Ready to fly] were also confirmed.
However, after connecting to the client, the beep sound does not stop from the driving unit.
Could you give me some guidance on this next step? or anyone who has built a Bigquad with the latest firmware can help? ;(
thank you

**Comment by ataffanel** [comment]
Tobias, our big-quad specialist, is currently on vacation so we will have to wait for him to come back to have more info, but I can try to help a bit and hopefully someone-else with big-quad experience will pass by :).

If does sounds like the ESCs are complaining of no communication or bad communication, you should look at the ESCs documentation to make sure what the beeping means.

To communicate with brushless ESCs, Crazyflie has support for PWM and multiple flavor of the DSHOT protocol, you should select the one adapted to your ESC.

Also make sure you are familiar with the arming system: https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/functional-areas/supervisor/arming/. In the latest client there is an arming button in the flight tab.
